src/qt/bitcoingui.cpp

```diff
@@ -118,6 +118,8 @@ BitcoinGUI::BitcoinGUI(interfaces::Node& node, const NetworkStyle* networkStyle,
         Q_EMIT consoleShown(rpcConsole);
     }
 
+    modalOverlay = new ModalOverlay(enableWallet, this->centralWidget());
```

Comment:
gui-30: Central widget is null at this point in our implementation. Also, we have no issues with the toolbar which original PR was aimed to fix. IMO, it's either DNM or merge changes in `src/qt/modaloverlay.h` only (except https://github.com/dashpay/dash/pull/5825/commits/b4c1af97d1d35cbb28feca6e3ae6705c25c01553#diff-bd2b12135c597b047fffc3b7a8e86e4a965c86ec7763f6ea885b6d8f6f702d10R40-R42 which is unused)

Reply:
Still not fixed. There should be no changes in `src/qt/bitcoingui.cpp` cause it breaks the way `modalOverlay` is shown (because of central widget being null at this point as I mentioned earlier).

Reply:
`setCentralWidget` is init just 5 lines above isn't i?

[src/qt/bitcoingui.cpp](https://github.com/dashpay/dash/pull/5825/files#diff-2ecf8cbf369cf3d2f3d2b1cf5cfe4c1a647d63e11e2885d2fd0ac11fb5f7a804)

Reply:
That's only when wallet support is not compiled in. When we do have wallet support compiled in `setCentralWidget ` is called inside of `createToolBars()`.